filters/include/pcl/filters/impl/filter_indices.hpp

```diff
@@ -74,7 +74,35 @@ pcl::removeNaNFromPointCloud (const pcl::PointCloud<PointT> &cloud_in,
   }
 }
 
+template<typename PointT> void
+pcl::FilterIndices<PointT>::applyFilter (PointCloud &output)
+{
+  std::vector<int> indices;
+  if (keep_organized_)
```

Comment:
Remember to skip this if `input_` is not organized.

Reply:
Opted to not change it know because it will trigger a behavior change that we might not want.